cobbler/modules/managers/bind.py

```diff
@@ -461,15 +461,17 @@ def __pretty_print_host_records(self, hosts, rectype: str = 'A', rclass: str = '
                 s += "%s  %s  %s  %s;\n" % (my_name, rclass, my_rectype, my_host)
         return s
 
-    def __pretty_print_cname_records(self, hosts, rectype: str = 'CNAME'):
+    def __pretty_print_cname_records(self, hosts, rectype: str = 'CNAME', rclass: str = 'IN'):
         """
         Format CNAMEs and with consistent indentation
 
         :param hosts: This parameter is currently unused.
         :param rectype: The type of record which shall be pretty printed.
+        :param rclass: The record class.
         :return: The pretty printed version of the cname records.
         """
         s = ""
+        zone_alliases = {}
```

Comment:
```suggestion
        zone_aliases = {}
```

I would like to fix that typo. (Of course, the rest of the code needs to rename the variable as well).